examples/programming-lang-detection/README.md

```diff
@@ -0,0 +1,80 @@
+# Programming Language Detection by MXNet 
+
+This example uses the power of MXNet alongside Keras to detect the programming language based upon an input file passed in.
+
+Currently, the @mxnet-label-bot uses this model to help both our contributors and new users by providing help which our users face
+in an appropriate manner.
+
+The script is able to take in a file name and detect the kind of programming language within the file as well as code snippets. Across the board we have found that it
+offers >90% accuracy for languages which we currently support of the file which is provided.
+## Usage ##
+
+Run:
+
+```
+python2.7 test.py <filename>
```

Comment:
please modify instructions - with the future lib import you can run this directly - `python test.py <file>`